ex03.exs

```diff
@@ -62,9 +62,14 @@ defmodule Ex03 do
   """
 
```

Comment:
I'm afraid this doesn't actually do anything in parallel: there are no calls to Task here...

Let's break down what to does by looking at the call

~~~ elixir
pmap(1..100, 4, &(&1+1))
~~~

This should divide the numbers from 1 to 100 up into four groups of 25. Each group should run in a separate process, and each process should return the result on incrementing its list of numbers. Finally the 4 results get joined back together to create the result.

There's nothing here that does the parallel part.

### -5